### PR TITLE
Releases page

### LINKED ISSUE #4103: Create the "Releases" tab in App Settings
# Desired Behavior

We want to use the backend we developed for #4102 to display a "Releases" tab in the app settings with the latest releases.

As on the website page, we want to parse every title (#, ##, ###)  into only one size: `(Base/semi-bold)` Bold style should be converted to `Base/medium`

image

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=23429-44442&mode=design&t=04MuXauFsKnLiI5R-11

### PR DESCRIPTION
**Pull request by brendanlaschke**
closes #4103 

Bildschirmfoto 2024-05-10 um 08 16 19

### CODE REVIEW

File: packages/twenty-website/src/app/api/releases/route.tsx [hunk 1]
```diff
@@ -42,7 +43,10 @@ export async function GET() {
       };
     });
 
-    return Response.json(formattedReleasesNotes);
+    const response = NextResponse.json(formattedReleasesNotes);
+    response.headers.append('Access-Control-Allow-Origin', '*');
```
Comment (brendanlaschke) [inline packages/twenty-website/src/app/api/releases/route.tsx:47]: For self hosted instances this is needed ... This shouldn't be a security problem?

Comment (FelixMalfait) [reply packages/twenty-website/src/app/api/releases/route.tsx:47]: Would you mind updating middleware.js to allow `*` directly on twenty-website? I think it was unnecessary to limit it (cc @ady-beraud fyi)

Comment (brendanlaschke) [reply packages/twenty-website/src/app/api/releases/route.tsx]: Sure done!

**Review by FelixMalfait — APPROVED** [review]
Great great contrib as always! Could you just check the width on mobile? It seems that something is making it larger than 100%

**Comment by brendanlaschke** [comment]
> Great great contrib as always! Could you just check the width on mobile? It seems that something is making it larger than 100%

It was a problem of the whole settings page - should now always be fitted on mobile screens. I hope this works in all sub menus :)

**Comment by FelixMalfait** [comment]
Ah yes indeed many settings pages look bad on mobile 😅

**Comment by FelixMalfait** [comment]
Thanks! 🚀

**Pull request merged**